openshift/e2e-tests-openshift.sh

```diff
@@ -83,83 +83,105 @@ function timeout() {
   return 0
 }
 
-function install_knative(){
-  header "Installing Knative serving"
+function install_servicemesh(){
+  header "Installing ServiceMesh"
 
-  create_knative_namespace serving
+  # Install the ServiceMesh Operator
+  oc apply -f openshift/servicemesh/operator-install.yaml
```

Comment:
I'd not vendor those files, if not really really needed:
https://github.com/openshift/knative-eventing/blob/release-v0.8.0/openshift/e2e-tests-openshift.sh#L54-L64